res/skins/QMLDemo/Mixxx/PlayerDropArea.qml

```diff
@@ -15,12 +15,15 @@ DropArea {
 
             console.log("Drag from group " + sourceGroup);
             player.cloneFromGroup(sourceGroup);
+            drop.accepted = true;
```

Comment:
this doesn't seems to be strictly relevant to this PR? Could you extract it into its own PR?

Reply:
No strictly, but it's relevant since the qml library implement drag and drop.

Reply:
https://github.com/mixxxdj/mixxx/pull/4067